crawl-ref/source/item-prop.cc

```diff
@@ -1999,6 +1999,9 @@ bool item_skills(const item_def &item, set<skill_type> &skills)
         skills.insert(SK_EVOCATIONS);
     }
 
+    if (is_unrandom_artefact(item, UNRAND_BATTLE))
+        skills.insert(SK_CONJURATIONS);
```

Comment:
Enhancer staves don't enable training of a skill. I know this one is a special case, since it brings a battlesphere, but the battle sphere isn't useful unless the player has other conj spells, in which case they'll be able to train conj just fine.